src/Router/appRoutes.js

```diff
@@ -15,6 +15,7 @@ const appRoutes = (
         <Route path={routes.ClientLogin} component={LoginPage} />
         <PrivateRoute exact path="/" component={HomePage} />
         <Route path={routes.ClientSignup} component={RegisterPage} />
+        <Route component={LoginPage} />
```

Comment:
What about the path for this component ?

Reply:
default route.If the user types some random address

Reply:
okay, I didn't see it wrapped in the switch component